express/garicarandai-2859/products/router.js

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import { productModel } from "./model.js";
+
+export const productRouter = express.Router();
+
+//CRUD: Create, Read, Update, Delete
+
+
+//Create with curl
+productRouter.post("/add-product", async (req, res)=>{
+  const products = await productModel.find({});
+  const short_id = products.length;
+  const product = {...req.body, short_id}
+  const createProduct = await productModel.create(product);
+  res.status(201).json({ product: createProduct });
+});
+
+// curl --request POST http://localhost:3001/products/add-product --header "Content-Type: application/json" --data '{ "product_name": "Test 1", "description": "Test 1.", "tags": ["t1.1", "t1.2", "t1.3"], "price": 30.00 }'
+
+
+
+//Read All with link / curl
+productRouter.get("/", async (req, res)=>{
+  const products = await productModel.find({})//.limit(10);
+  res.status(200).json({ products: products });
```

Comment:
```suggestion
  res.status(200).json({ products });
```
Cuando el atributo y la variable que tiene el valor que quieres asignar tienen el mismo nombre, puedes declararlo de esta forma. No es neesario que lo corrigas, igual esta bien